contracts/handler/CompatibilityFallbackHandler.sol

```diff
@@ -9,6 +9,8 @@ import "../GnosisSafe.sol";
 /// @title Compatibility Fallback Handler - fallback handler to provider compatibility between pre 1.3.0 and 1.3.0+ Safe contracts
 /// @author Richard Meissner - <richard@gnosis.pm>
 contract CompatibilityFallbackHandler is DefaultCallbackHandler, ISignatureValidator {
+    bytes4 internal constant SIMULATE_SELECTOR = bytes4(keccak256("simulate(address,bytes)"));
```

Comment:
Why not hardcoded? It's already translated by the compiler? Then why on other places we are using the hardcoded version of a keccak256 hash

Reply:
Runtime gas costs are not affected only the deployment gas costs ... and I did not care about them here (as the contract is far from the limit). This way it would be easier to adjust if necessary.